pyatlan/model/credential.py

```diff
@@ -94,6 +94,16 @@ def to_credential(self) -> Credential:
         )
 
 
+class CredentialResponseList(AtlanObject):
+    """
+    Model representing a response containing a list of CredentialResponse objects.
+
+    Attributes:
+        records (List[CredentialResponse]): The list of credential records returned.
+    """
+    
+    records: List[CredentialResponse] = Field(..., description="The list of credential records returned.")
```

Comment:
Let's always use `Optional` with `default=None` to avoid any Pydantic validation errors based on model field strict constraints:

```suggestion
    records: Optional[List[CredentialResponse]] = Field(default=None, description="list of credential records returned.")
```